src/main/java/de/bonndan/nivio/ProcessingFinishedEvent.java

```diff
@@ -22,11 +28,18 @@ public ProcessingFinishedEvent(LandscapeDescription source, Landscape landscape)
         this.landscape = landscape;
     }
 
-    @Override
+    @JsonSerialize(using = PLS.class)
     public Landscape getLandscape() {
         return landscape;
     }
 
```

Comment:
Yes. We might refactor that, too. It uses both (in "source" and "landscape"). The landscape is needed by the ObserverRegistry at the moment to let it know which landscape is subject to change. However, it could be a mistake of thought and we might get the same behaviour with the LandscapeDescription using its FullyQualifiedIdentifier.